components/contribution-flow/ContributorCardWithTier.js

```diff
@@ -17,6 +18,8 @@ import { P } from '../Text';
 import { withUser } from '../UserProvider';
 
 const ContributorCardWithTier = ({ contribution, ...props }) => {
+  useEffect(() => atcb_init());
```

Comment:
This will call `atcb_init` on each component update.
- Shouldn't we call it only on mount?
- Shouldn't we call it only when the contribution was made to an event?